tests/pytest/enrollment/test_views.py

```diff
@@ -194,14 +250,358 @@ def test_index_eligible_post_valid_form_success(
     response = client.post(path, card_tokenize_form_data)
 
     mock_client.link_concession_group_funding_source.assert_called_once_with(
-        funding_source_id=mocked_funding_source.id, group_id=model_EligibilityType.group_id
+        funding_source_id=mocked_funding_source.id, group_id=model_EligibilityType.group_id, expiry_date=None
```

Comment:
Maybe this test function should be renamed to `test_index_eligible_post_valid_form_success_no_expiry` or something?

Reply:
Yeah, I thought about that too and am ok with updating the name. I think then we should also rename things like `model_EligibilityType`, `mocked_funding_source`, etc. to say `no_expiry`. Any objection to that?

Reply:
Sounds good!

Reply:
Renamed tests/fixtures in these two commits: https://github.com/cal-itp/benefits/pull/1989/files/15bf0c9bb93fbc0a8e80a905c8422f5efbaaeb98..9b157659ec748908ca70d2d2077c283587bfe7e8